### PR TITLE
Update release schedule by moving 2 weeks for end of year holidays

### PR DESCRIPTION
**Pull request by nak3**
This patch update release schedule by moving 2 weeks for end of year holidays.

It is now sync with https://github.com/knative/community/blob/master/mechanics/RELEASE-SCHEDULE.md

/cc @tcnghia @evankanderson @slinkydeveloper

### CODE REVIEW

File: RELEASE-LEADS.md
Comment (nak3) [inline RELEASE-LEADS.md:51]: I didn't change `PKG cut` as `2020-12-15`. It should be updated? @evankanderson @tcnghia

Comment (slinkydeveloper) [inline RELEASE-LEADS.md:51]: 2020-01-05?

Comment (slinkydeveloper) [reply RELEASE-LEADS.md:51]: 2020-01-06 it's a religious holiday in some eu places, can we move it to 2020-01-07?

**Review by slinkydeveloper — COMMENTED** [review]
left some comments about the date

File: RELEASE-LEADS.md
Comment (tcnghia) [reply RELEASE-LEADS.md]: Since you and @slinkydeveloper will be handle that release, I think you two should decide how best to proceed there. However a lot of that has been automated by buoy

cc @n3wscott

Comment (nak3) [reply RELEASE-LEADS.md]: I see. Thank you.
@slinkydeveloper I am fine with 2020-12-15 pkg cut. Please let me know if you have any objections.

Comment (nak3) [reply RELEASE-LEADS.md]: @slinkydeveloper As per [the instruction](https://github.com/knative/pkg/blob/master/RELEASE-LEADS.md#1-day-prior-to-the-release), `1 day prior to the release` and `Day of the release` have some tasks, so 2021-01-05 is better than 2021-01-07?
We may have extra task if there are any troubles on `2021-01-06` :smile: but the US folks are supposed to work on that day.

**Review by tcnghia — APPROVED** [review]

File: RELEASE-LEADS.md
Comment (slinkydeveloper) [reply RELEASE-LEADS.md]: > We may have extra task if there are any troubles on 2021-01-06 smile but the US folks are supposed to work on that day.

That's my worry, i won't be online for sure on 01-06 because it's holiday in italy and i'm worried that eu folks might be offline the 2 days before too (4 and 5)

File: RELEASE-LEADS.md
Comment (n3wscott) [inline RELEASE-LEADS.md:51]: The week before a release, the release managers need to be hands on keyboards cutting branches, in the new world. I would push the release one more week to allow for this. Otherwise the release will not go smoothly.

Comment (nak3) [reply RELEASE-LEADS.md:51]: Hi @vagababov @grantr @tcnghia @evankanderson  I am tagging you as the schedule was decided by https://github.com/knative/community/pull/226.
Is it alright to push one more week for the v0.20 release schedule?

Comment (evankanderson) [reply RELEASE-LEADS.md:51]: Good question -- I have no intrinsic problem with doing this; the previous date extension was decided with the notion that a release (and subsequent hangover of e.g. docs and operator at the moment) should have hands on deck after release in case of bugs.

If we need hands on deck before release (and can't relax that by e.g. pre-cutting the `pkg` branch of other prep), then moving another week seems okay.

Comment (nak3) [reply RELEASE-LEADS.md]: Okay, pushed one more week.  
@slinkydeveloper  we will do pkg cut on `2020-01-05` and release on `2020-01-12`. Is that alright?

**Comment by slinkydeveloper** [comment]
Why this was merged? The agreement was not reached yet :smile:

File: RELEASE-LEADS.md
Comment (slinkydeveloper) [reply RELEASE-LEADS.md]: Can we do pkg cut on `2020-01-07` so we're after the epiphany? I guess we won't have big contributions to pkg in that period, 5 days of freeze more than one week is fine i think. Or is that a "strong" requirement to be one week before?

Comment (nak3) [reply RELEASE-LEADS.md]: @slinkydeveloper Sure, I'm fine with `2020-01-07 `.  Could you send the PR to update the pkg cut schedule?

**Comment by n3wscott** [comment]
> Why this was merged? The agreement was not reached yet 😄

so sorry, I thought it was...

**Comment by slinkydeveloper** [comment]
NP :+1:

**Pull request merged**
